src/SFA.DAS.AdminService.Web.UnitTests/Controllers/Home/HomeControllerTest.cs

```diff
@@ -49,26 +52,48 @@ public void Then_Index_Returns_ViewModel(bool useDfESignIn)
         public void When_DfESignIn_True_And_User_Authenticated_Then_Index_Returns_Redirect()
         {
             //arrange
-            _configuration.Setup(c => c.UseDfESignIn).Returns(true);
+            _mockWebConfiguration.Setup(c => c.UseDfESignIn).Returns(true);
             var authorizedUser = new ClaimsPrincipal(new ClaimsIdentity(new[]
             {
                 new Claim("name", "somename")
             }, "mock"));
             var httpContext = new Mock<HttpContext>();
             httpContext.Setup(c => c.User).Returns(authorizedUser);
-            var controller = new HomeController(_configuration.Object)
+            _controller = new HomeController(_mockWebConfiguration.Object, _mockConfiguration.Object)
             {
                 ControllerContext = new ControllerContext { HttpContext = httpContext.Object }
             };
 
             //sut
-            var result = controller.Index();
+            var result = _controller.Index();
 
             //assert
             result.Should().NotBeNull();
             var actualResult = (RedirectToActionResult)result;
             actualResult.ControllerName.Should().Be("Dashboard");
             actualResult.ActionName.Should().Be("Index");
```

Comment:
If one of these asserts fail, the test will abort without running the subsequent asserts. We can avoid this by wrapping this block in an `AssertionScope`
```suggestion
           using (new AssertionScope())
           {
                result.Should().NotBeNull();
                var actualResult = (RedirectToActionResult)result;
                actualResult.ControllerName.Should().Be("Dashboard");
                actualResult.ActionName.Should().Be("Index");
           }
```